### ISSUE TITLE
semi working

### ISSUE THREAD

**Issue by stainlessdk** [dotsam/homebridge-milight#35]
Hi

im having trouble changeing the color with the siri commands, its working fine in the homeasistent, it keeps saying its not supported

i have a v5 and v6 (ibox2) brigde

my bulbs are gu10 rgbw (not cct model) i cant figure out if the problem is with homebridge or milight plugins

any advice 

here is my config

 "ip_address": "172.16.10.227",
                "version": "v6",
                "lights": {"fullColor": ["Kitchen",null,"Bedroom","Hallway"],
                           "rgbw": ["Living Room", "Downstairs Bedroom"]},
                "repeat": 1,
                "delay": 100

/ Stainless

**Comment by dotsam** [comment]
Hi Stainless,

Any more details on exactly what the error is here? What command do you issue to Siri, and what's the response? Are you able to change colours through the Home app?

**Comment by toshibochan** [comment]
Try.
One bridge can do 4 zone

"lights": {"rgbw": ["Living Room", "Downstairs", "Bedroom", "kitchen"]},
                "repeat": 1,
                "delay": 100

**Comment by toshibochan** [comment]
"lights": {"rgbw": ["zone1", "zone2", "zone3", "zone4"]},

**Comment by stainlessdk** [comment]
"ip_address": "172.16.10.215",
                        "version": "v6",
                        "lights": {"rgbw": ["udelys1", "gang", "laerke"]},
                        "repeat": 1,
                        "delay": 100

i have tried this, and its still the same. i can change colour from the home assistent manually. but when i tell siri she says its not supported, also when im using the dimmer when it comes down to 1% the light changes to orange

**Comment by stainlessdk** [comment]
im telling siri to change color gang to red

**Comment by toshibochan** [comment]
You can use home app to control the light on iOS?

**Comment by toshibochan** [comment]
You sure you Mi-Light hub is v6?

**Comment by toshibochan** [comment]
Try Change repeat to 3

**Comment by stainlessdk** [comment]
yes i can change color in the app control, the hub im using is 

https://www.aliexpress.com/item/Milight-WiFi-iBox2-led-Controller-USB-Cable-Wireless-Dimmer-Controller-For-Mi-Light-Series-RGBW-WW/32783130404.html?spm=2114.search0104.3.17.KbIFKo&ws_ab_test=searchweb0_0,searchweb201602_3_10152_10065_10151_10344_10068_10345_10342_5660015_10343_10340_10341_10171_10541_10084_10083_10304_10307_10302_10060_10155_10154_10056_10055_10539_10537_10312_10313_10536_10059_10314_10534_10533_100031_5650015_10550_10103_10073_10551_10102_10552_5670015_10553_10554_10557_10142_10107-10552,searchweb201603_17,ppcSwitch_2&btsid=e5017237-8567-4c84-91c4-565c61dc3411&algo_expid=f4876a52-84cf-470d-9fe4-a4f68baa2f68-2&algo_pvid=f4876a52-84cf-470d-9fe4-a4f68baa2f68

**Comment by stainlessdk** [comment]
Try Change repeat to 3 <-still the same, it changes to orange / red when dimming

**Comment by toshibochan** [comment]
I don’t know why for control color with Siri on v6 hub, color not correct!

**Comment by toshibochan** [comment]
I have color issues to! And I don’t know why but probably is Mi-Light hub problem

**Comment by stainlessdk** [comment]
but can you change color with siri, or only on the home app ?

**Comment by toshibochan** [comment]
I can use Siri yes

**Comment by toshibochan** [comment]
You have other devices you can control with Siri?

**Comment by stainlessdk** [comment]
yes i have wiwo outlet, thats also working. what do you tell siri when you want to change color, i can turn on and off the light in my house but no color change

**Comment by toshibochan** [comment]
Did you set the room? Try to add room and put the light in the room.
Exemple:
Light name: kitchen ceiling light
Room name: Kitchen 
Hey Siri set the kitchen light to red
Hey Siri set kitchen ceiling light to red

**Comment by stainlessdk** [comment]
Hey Sorry for the late reply, been working my a.. off, t

its working now :1st_place_medal: 

"but" only on my v6 hub, my old v5 dosent support it, but thats ok :)

**Comment by stainlessdk** [comment]
but still when im dimming the orange issue returns, but siri commands works so im happy
